<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>20</y>
      <width>171</width>
      <height>531</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="displayFormat">
        <string>MM/dd/yyyy h:mm AP</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="checkBoxObstacle">
       <property name="text">
        <string>Obstacle Ahead</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QComboBox" name="comboBoxWeather">
       <item>
        <property name="text">
         <string>Clear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Fog</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rain</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QPushButton" name="buttonLog">
       <property name="text">
        <string>Show Log</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QSpinBox" name="spinBoxSpeed">
       <property name="suffix">
        <string> mph</string>
       </property>
       <property name="maximum">
        <number>200</number>
       </property>
       <property name="value">
        <number>30</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QPushButton" name="buttonSimulate">
       <property name="text">
        <string>Simulate Decision</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QComboBox" name="comboBoxRoadType">
       <item>
        <property name="text">
         <string>City</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Highway</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="checkBoxPedestrian">
       <property name="text">
        <string>Pedestrian Nearby</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QTextBrowser" name="textBrowserLog"/>
     </item>
     <item row="16" column="0">
      <widget class="QPushButton" name="buttonReset">
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelVideo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>511</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSRC_Project">
    <property name="title">
     <string>SRC Project</string>
    </property>
   </widget>
   <addaction name="menuSRC_Project"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
